### DISCUSSION TITLE
Video Streaming can be slow

### DISCUSSION THREAD

**Discussion by atejeda08** [tesla-android/issue-tracker#266] [Q&A]
I stream quite a bit of video. While youtube app works wonderfully, I struggle to get other apps like Prime Video (where I watch live sports mostly) to stream without dropping video (never audio) every 10 seconds or so. 

My question is: is this just due to hardware/software limitations of this awesome project? Or is there something I could do to help my case. I use it with USB tethering and very rarely my iphone isn't in 5G cell coverage. 

Thanks in advance for any help.

**Comment by mikegapinski** [comment]
I know that prime video works perfectly fine, but they don't provide live tv in the EU so I only watch prerecorded content. Any clue about the codec they use?
